diff --git a/node_modules/@lerna/version/command.js b/node_modules/@lerna/version/command.js
index 974ce66..dab9e06 100644
--- a/node_modules/@lerna/version/command.js
+++ b/node_modules/@lerna/version/command.js
@@ -52,6 +52,11 @@ exports.builder = (yargs, composed) => {
       requiresArg: true,
       defaultDescription: "origin",
     },
+    "since": {
+      describe: "Specify a particular commit for diffing with the previous version",
+      type: "string",
+      requiresArg: true
+    },
     "create-release": {
       describe: "Create an official GitHub or GitLab release for every version.",
       type: "string",
