diff --git a/node_modules/fast-deep-equal/index.js b/node_modules/fast-deep-equal/index.js
index 27264f5..133fc13 100644
--- a/node_modules/fast-deep-equal/index.js
+++ b/node_modules/fast-deep-equal/index.js
@@ -4,7 +4,7 @@ var isArray = Array.isArray;
 var keyList = Object.keys;
 var hasProp = Object.prototype.hasOwnProperty;
 
-module.exports = function equal(a, b) {
+export default function equal(a, b) {
   if (a === b) return true;
 
   if (a && b && typeof a == 'object' && typeof b == 'object') {
