diff --git a/node_modules/racer-highway/lib/browser/socket.js b/node_modules/racer-highway/lib/browser/socket.js
index 4a08263..6b8d6d9 100644
--- a/node_modules/racer-highway/lib/browser/socket.js
+++ b/node_modules/racer-highway/lib/browser/socket.js
@@ -1,6 +1,6 @@
 module.exports = Socket;
 
-var BCSocket = require('browserchannel/dist/bcsocket-uncompressed').BCSocket;
+// var BCSocket = require('browserchannel/dist/bcsocket-uncompressed').BCSocket;
 
 function Socket(options) {
   this._options = options;
@@ -9,11 +9,11 @@ function Socket(options) {
   this._attemptNum = 0;
   this._url = getWebSocketURL(options);
 
-  if (supportWebSockets() && !options.browserChannelOnly) {
+  // if (supportWebSockets() && !options.browserChannelOnly) {
     this._createWebSocket();
-  } else {
-    this._createBrowserChannel();
-  }
+  // } else {
+  //   this._createBrowserChannel();
+  // }
 }
 
 Socket.prototype._createWebSocket = function() {
@@ -30,21 +30,21 @@ Socket.prototype._createWebSocket = function() {
 
 };
 
-Socket.prototype._createBrowserChannel = function() {
-  this._type = 'browserchannel';
-  this._socket = BCSocket(this._options.base, this._options);
+// Socket.prototype._createBrowserChannel = function() {
+//   this._type = 'browserchannel';
+//   this._socket = BCSocket(this._options.base, this._options);
 
-  this.open = this._createBrowserChannel.bind(this);
-  this._syncState();
+//   this.open = this._createBrowserChannel.bind(this);
+//   this._syncState();
 
-  this._socket.onmessage = this._bc_onmessage.bind(this);
-  this._socket.onopen = this._bc_onopen.bind(this);
-  this._socket.onclose = this._bc_onclose.bind(this);
-};
+//   this._socket.onmessage = this._bc_onmessage.bind(this);
+//   this._socket.onopen = this._bc_onopen.bind(this);
+//   this._socket.onclose = this._bc_onclose.bind(this);
+// };
 
 Socket.prototype._ws_onmessage = function(message) {
   this._syncState();
-  message.data = JSON.parse(message.data);
+  // message.data = JSON.parse(message.data);
   this.onmessage && this.onmessage(message);
 };
 
@@ -64,9 +64,9 @@ Socket.prototype._ws_onclose = function(event) {
 
   this.onclose && this.onclose(event);
 
-  if (!this._connectedOnce) {
-    return this._createBrowserChannel();
-  }
+  // if (!this._connectedOnce) {
+  //   return this._createBrowserChannel();
+  // }
 
   var socket = this;
 
