diff --git a/node_modules/@startupjs/racer/lib/Model/events.js b/node_modules/@startupjs/racer/lib/Model/events.js
index aa58be9..ab8a4b9 100644
--- a/node_modules/@startupjs/racer/lib/Model/events.js
+++ b/node_modules/@startupjs/racer/lib/Model/events.js
@@ -241,7 +241,7 @@ Model.prototype.removeContextListeners = function(value) {
  * @param {string} eventType
  */
 function eventListener(model, eventType, arg2, arg3, arg4) {
-  var subpattern, options, cb;
+  var subpattern, options, cb, useEventObjects;
   if (arg4) {
     // on(eventType, path, options, cb)
     subpattern = arg2;
@@ -262,7 +262,7 @@ function eventListener(model, eventType, arg2, arg3, arg4) {
   }
   if (options) {
     if (options.useEventObjects) {
-      var useEventObjects = true;
+      useEventObjects = true;
     }
   }
 
diff --git a/node_modules/@startupjs/racer/lib/Model/index.js b/node_modules/@startupjs/racer/lib/Model/index.js
index 07101f5..0413d1d 100644
--- a/node_modules/@startupjs/racer/lib/Model/index.js
+++ b/node_modules/@startupjs/racer/lib/Model/index.js
@@ -20,7 +20,7 @@ import './refList';
 import './ref';
 
 // Extend model for server //
-if (util.isServer) import('./bundle');
-if (util.isServer) import('./connection.server');
+// if (util.isServer) import('./bundle');
+// if (util.isServer) import('./connection.server');
 
 export default Model;
diff --git a/node_modules/@startupjs/racer/lib/Racer.js b/node_modules/@startupjs/racer/lib/Racer.js
index 0c2909c..1fb1035 100644
--- a/node_modules/@startupjs/racer/lib/Racer.js
+++ b/node_modules/@startupjs/racer/lib/Racer.js
@@ -27,4 +27,4 @@ Racer.prototype.createModel = function(data) {
   return model;
 };
 
-if (util.isServer) import('./Racer.server');
+// if (util.isServer) import('./Racer.server');
