diff --git a/node_modules/react-router-native-stack/lib/Stack.js b/node_modules/react-router-native-stack/lib/Stack.js
index 805d13a..bc225f0 100644
--- a/node_modules/react-router-native-stack/lib/Stack.js
+++ b/node_modules/react-router-native-stack/lib/Stack.js
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
-import { Platform, View, ViewPropTypes } from 'react-native';
-import { node, object, bool, string, func, oneOf } from 'prop-types';
+import { Platform, View } from 'react-native';
+import { node, object,array, bool, string, func, oneOf,oneOfType } from 'prop-types';
 import { withRouter } from 'react-router-native';
 import StackTransitioner from './StackTransitioner';
 import { PUSH, POP } from './transitionTypes';
@@ -18,7 +18,7 @@ class Stack extends Component {
     renderRightSegment: func,
     animationType: string,
     gestureEnabled: bool,
-    stackViewStyle: ViewPropTypes.style,
+    stackViewStyle: oneOfType([object, array]),
     replaceTransitionType: oneOf([PUSH, POP]),
     isAnimating: func,
   };
diff --git a/node_modules/react-router-native-stack/lib/StackTransitioner.js b/node_modules/react-router-native-stack/lib/StackTransitioner.js
index 1c170fe..94f7699 100644
--- a/node_modules/react-router-native-stack/lib/StackTransitioner.js
+++ b/node_modules/react-router-native-stack/lib/StackTransitioner.js
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
-import { node, object, number, bool, string, oneOf, func } from 'prop-types';
-import { Animated, PanResponder, View, ViewPropTypes } from 'react-native';
+import { node, object, array, number, bool, string, oneOf, func, oneOfType } from 'prop-types';
+import { Animated, PanResponder, View } from 'react-native';
 import findFirstMatch from './findFirstMatch';
 import getTransforms from './getTransforms';
 import getEasing from './getEasing';
@@ -24,7 +24,7 @@ export default class StackTransitioner extends Component {
     height: number.isRequired,
     animationType: string.isRequired,
     gestureEnabled: bool,
-    stackViewStyle: ViewPropTypes.style,
+    stackViewStyle: oneOfType([object, array]),
     replaceTransitionType: oneOf([PUSH, POP]),
     isAnimating: func
   };
