diff --git a/node_modules/react-native-web/dist/cjs/exports/StyleSheet/createStyleResolver.js b/node_modules/react-native-web/dist/cjs/exports/StyleSheet/createStyleResolver.js
index fc4a126..006e447 100644
--- a/node_modules/react-native-web/dist/cjs/exports/StyleSheet/createStyleResolver.js
+++ b/node_modules/react-native-web/dist/cjs/exports/StyleSheet/createStyleResolver.js
@@ -365,7 +365,7 @@ var createCacheKey = function createCacheKey(id) {
 };

 var classListToString = function classListToString(list) {
-  return list.join(' ').trim();
+  return list.join(' ').trim().replace(/(^| )undefined( |$)/g, ' ')
 };
 /**
  * Copies classList and style data from a DOM node
diff --git a/node_modules/react-native-web/dist/cjs/exports/Text/index.js b/node_modules/react-native-web/dist/cjs/exports/Text/index.js
index 3d353e8..71ac9b7 100644
--- a/node_modules/react-native-web/dist/cjs/exports/Text/index.js
+++ b/node_modules/react-native-web/dist/cjs/exports/Text/index.js
@@ -78,7 +78,7 @@ function (_Component) {
       otherProps.onKeyDown = this._createEnterHandler(onPress);
     }

-    otherProps.classList = [this.props.className, classes.text, this.context.isInAParentText === true && classes.textHasAncestor, numberOfLines === 1 && classes.textOneLine, numberOfLines > 1 && classes.textMultiLine]; // allow browsers to automatically infer the language writing direction
+    otherProps.classList = [classes.text, this.context.isInAParentText === true && classes.textHasAncestor, numberOfLines === 1 && classes.textOneLine, numberOfLines > 1 && classes.textMultiLine]; // allow browsers to automatically infer the language writing direction

     otherProps.dir = dir !== undefined ? dir : 'auto';
     otherProps.style = [style, numberOfLines > 1 && {
diff --git a/node_modules/react-native-web/dist/cjs/exports/View/index.js b/node_modules/react-native-web/dist/cjs/exports/View/index.js
index 61d92b6..c5dac80 100644
--- a/node_modules/react-native-web/dist/cjs/exports/View/index.js
+++ b/node_modules/react-native-web/dist/cjs/exports/View/index.js
@@ -68,7 +68,7 @@ function (_Component) {
     }

     var isInAParentText = this.context.isInAParentText;
-    supportedProps.classList = [this.props.className, classes.view];
+    supportedProps.classList = [classes.view];
     supportedProps.style = _StyleSheet.default.compose(isInAParentText && styles.inline, this.props.style);

     if (hitSlop) {
